test/test-meshctl-status.js

```diff
@@ -36,8 +36,8 @@ test('Test status command', function(t) {
     'Environment variables:',
     '  No environment variables defined',
     'Instances:',
-    '    Version  Agent version  Cluster size',
-    '     1.2.3       7.8.9            3',
+    '    Version  Agent version  Cluster size  Driver metadata',
+    '     1.2.3       7.8.9            3              {}',
```

Comment:
I'm working on a PR for PM that will result in the docker driver printing something like `{container: '12ab34cd56ef'}`